### PR TITLE
Fix multi-user doc typo

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Fix multi-user doc typo

## Motivation and Context
Fixes a typo

## Regression
No

## How Has This Been Tested?
Fixes typo in the doc and improve upon existing one a bit. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/multi-user/README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ In this document we will explain in detail on how to configure multiple users.
 - Install mc - [Minio Client Quickstart Guide](https://docs.minio.io/docs/minio-client-quickstart-guide.html)
 - Install Minio - [Minio Quickstart Guide](https://docs.minio.io/docs/minio-quickstart-guide)
 
-### 2. Create a new user and policy
-Create new canned policy `getonly` with `newuser.json` use `mc admin policies`. This policy enables users to download all objects in my-bucketname.
+### 2. Create a new user with canned policy
+Use [`mc admin policies`](https://docs.minio.io/docs/minio-admin-complete-guide.html#policies) to create canned policies. This policy enables users to download all objects under `my-bucketname`.
```
Comment (poornas) [inline docs/multi-user/README.md]: > This policy enables users to download all objects under `my-bucketname`.

This line seems out of place here. should follow after ```Create new...```

**Review by poornas — CHANGES_REQUESTED** [review]

File: docs/multi-user/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/multi-user/README.md]: fixed

File: docs/multi-user/README.md [hunk 2]
```diff
@@ -49,6 +54,12 @@ Remove the user `newuser`.
 mc admin users remove myminio newuser
 ```
 
+### 5. Change user policy
+Change the policy for user `newuser` to `putonly` canned policy.
+```
+mc admin user policy myminio newuser putonly
```
Comment (poornas) [inline docs/multi-user/README.md]: user -> users

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :x: [more...](#1145d77a-48fb-4e21-bdc9-d2778373ad80) |
<a name="1145d77a-48fb-4e21-bdc9-d2778373ad80"></a>
### 6662-fa11295/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30726
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 19d5398b7d60:/mint/log /tmp/mint-logs'
(1/13) Running awscli tests ... FAILED in 25 seconds
{
  "name": "awscli",
  "duration": 12121,
  "function": "aws --endpoint-url http://minikube:30726 s3 cp /mint/data/datafile-65-MB s3://awscli-mint-test-bucket-8202/datafile-65-MB\n",
  "status": "FAIL",
  "error": "upload failed: ../../../data/datafile-65-MB to s3://awscli-mint-test-bucket-8202/datafile-65-MB An error occurred (XMinioServerNotInitialized) when calling the CreateMultipartUpload operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: docs/multi-user/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# Minio multi-user Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
-This document explains how to add, revoke users. Multi-user as name implies means Minio supports long term users other than default credentials, each of these users can be configured to deny or allow access to buckets, resources.
+# Minio Multi-user Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+This document explains how to add, disable users. Multi-user as name implies means Minio supports long term users other than default credentials, each of these users can be configured to deny or allow access to buckets, resources.
```
Comment (nitisht) [inline docs/multi-user/README.md]:
```suggestion
Minio supports multiple long term users in addition to default user created during server startup. New users can be added after server starts up, and server can be configured to deny or allow access to buckets and resources to each of these users. This document explains how to add/remove users and modify their access rights.

```

**Review by nitisht — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
Ping @nitisht @poornas

**Review by nitisht — APPROVED** [review]

**Review by poornas — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Pull request merged**